channel.go

```diff
@@ -264,8 +264,8 @@ type ExternalMediaOptions struct {
 	// Direction specifies the directionality of the audio stream.  Options include 'both'.  This parameter is optional and if not specified, 'both' will be used.
 	Direction string `json:"direction"`
 
-	// Variables defines the set of channel variables which should be bound to this channel upon creation.  This parameter is optional.
-	Variables map[string]string `json:"variables"`
```

Comment:
Is there any need to remove `Variables` from `ExternalMediaOptions` struct and pass them as argument ?
Also `data` field was introduced with Asterisk 18, there is no such field in Asterisk 16 and its EOL is 2023-10-09.

I propose to ignore `data` field for now leaving struct as it is and only fix handling of `ExternalMediaOptions` for query.